ext/com_dotnet/com_com.c

```diff
@@ -720,7 +728,9 @@ PHP_FUNCTION(com_event_sink)
 		if ((tmp = zend_hash_index_find(Z_ARRVAL_P(sink), 1)) != NULL && Z_TYPE_P(tmp) == IS_STRING)
 			dispname = Z_STRVAL_P(tmp);
 	} else if (sink != NULL) {
-		convert_to_string(sink);
+		if (!try_convert_to_string(sink)) {
+			return;
```

Comment:
As these are in a PHP_FUNCTION you could use ``RETURN_THROWS()``

Reply:
Ah, yes, thanks! I'll do so when merging into master (PR targets PHP 7.4).

Reply:
Right I didn't see the target :)